src/llvm-late-gc-lowering.cpp

```diff
@@ -828,6 +828,27 @@ std::vector<int> LateLowerGCFrame::NumberAllBase(State &S, Value *CurrentV) {
                 Numbers.push_back(BaseNumbers[i]);
         }
         assert(CountTrackedPointers(EVI->getType()).count == Numbers.size());
+    } else if (auto *II = dyn_cast<IntrinsicInst>(CurrentV)) {
+        switch (II->getIntrinsicID()) {
+            case Intrinsic::masked_gather: {
+                Numbers = NumberAll(S, II->getOperand(0));
```

Comment:
I am simply forwarding the operand here, unsure if that is the right way of doing things, additionally should we handle
other masked intrinsics like `masked.scatter` and `masked.load`?

According to the Langref https://llvm.org/docs/LangRef.html#llvm-masked-gather-intrinsics there is also a passthrough
argument which would be a tracked pointer (or in this case it was undef).

Reply:
No, it’s equivalent to a load, so they’re both a normal generating instruction (just like a regular CallInst)